go.mod

```diff
@@ -1,17 +1,17 @@
 module github.com/buildkite/cli
 
+go 1.11
```

Comment:
I imagine the change from `go 1.13` (bottom of file) to `go 1.11` here is unintentional?
Interestingly, when I run `go mod tidy` against `master` using go 1.14, it leaves the go 1.13 line untouched.

I guess go 1.11 is coming from the `image: "golang:1.11"` being passed to the docker plugin, which does match the version being passed to `golang-cross-compile#v1.3.0` for the build.

Maybe I'll sneak in a go 1.14 upgrade which we can rebase this PR onto before merging, so that we don't downgrade go.mod from 1.13 to 1.11 and then back up to 1.14.

Reply:
I am not how far you are aware but the `go` directive in `go.mod` does not force a specific version of the Go compiler, it simply gives a compatibility indication with respect to the use of features of newer Go releases. As a result this being at 1.11 does not really matter, in many regards.